pkgs/misc/vim-plugins/generated.nix

```diff
@@ -7163,7 +7175,7 @@ final: prev:
       owner = "flazz";
       repo = "vim-colorschemes";
       rev = "fd8f122cef604330c96a6a6e434682dbdfb878c9";
-      sha256 = "1cg8q7w0vgl73aw1b9zz0zh5vw5d2pm8pm54fhfzva4azg56f416";
+      sha256 = "0kpsf6j20fgblc8vhqn7ymr52v2d1h52vc7rbxmxfwdm80nvv3g5";
```

Comment:
This causes a hash mismatch, also see as https://github.com/NixOS/nixpkgs/pull/145640#issuecomment-971126083
somehow this is the exact same hash, is there something wrong with the update script?

cc @dotlambda @lovesegfault @chuahou from the previous conversations

Reply:
:ghost:

Reply:
This is really strange... @dotlambda and I also had very different results running update.py minutes apart from each other. It shouldn't be different nixpkgs versions since I'm assuming we're all running it on master, and `update-shell.nix` refers to the current nixpkgs ref's packages, so perhaps it is something to do with the GitHub access token used? (very wild guess)

Reply:
Something funky is going on with how vim-colorschemes is being fetched. I get [the same hash mismatch](https://github.com/NixOS/nixpkgs/pull/146342#issuecomment-971335015) when building it from this commit myself, but ofborg [had no such trouble](https://gist.github.com/GrahamcOfBorg/3e971c938a1ddbfc15574d64226f7031).

Reply:
Running update.py again on top of this commit reverts the sha256 change, as in line 7178 of https://github.com/ch-fork/nixpkgs/commit/a7b84f9843d2f3dd8489ec11ca1c6b37f8991adc.

Reply:
The word `Update` is capitalized because I followed these instructions https://nixos.wiki/wiki/Vim#Adding_new_plugins
And indeed initially I ran update.py with `--no-commit` because I wanted to inspect the changes and do it manually to learn how it works